radio/src/gui/common/stdlcd/radio_hardware.cpp

```diff
@@ -558,6 +558,12 @@ void menuRadioHardware(event_t event)
 #if defined(BLUETOOTH)
       case ITEM_RADIO_HARDWARE_BLUETOOTH_MODE:
         lcdDrawTextAlignedLeft(y, STR_BLUETOOTH);
+#if defined(PCBX9E)
+        if (g_model.trainerData.mode == TRAINER_MODE_MASTER_SBUS_EXTERNAL_MODULE) {
```

Comment:
It remains an issue if you have BT enabled and you load a model with SBUS TRAINER enabled. In this case you will have BT enabled and impossible to remove it.